bower.json

```diff
@@ -14,5 +14,20 @@
     "angular-ui-router": "~0.2.18",
     "bootstrap": "~3.3.6",
     "owasp-password-strength-test": "~1.3.0"
+  },
+  "overrides": {
+    "bootstrap": {
+      "main": [
+        "dist/css/bootstrap.css",
+        "dist/css/bootstrap-theme.css",
+        "less/bootstrap.less"
+      ]
+    },
+    "jquery": {
+      "main": []
+    }
+  },
+  "resolutions": {
+    "angular": "~1.3"
```

Comment:
@ilanbiala What should we do about this Angular version resolution? I know we recently removed it from `bower.json`, when we upgraded to Angular 1.5. However, it seems that we still have this issue. On *master* I'm prompted to select a version of Angular to resolve to when I install the Bower dependencies. 

Do we want to keep this here? If so, I think it should be set to `~1.5`. WDYT?